wiki/en/misc/1-index.html

```diff
@@ -30,26 +30,25 @@ <h2>What is Jamulus?</h2>
 </div>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
-     <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
-    in schools and universities, in private and in public. Even just for the spoken word – all in real time on the Internet, as if you were there in person.
-    <h2>Help needed?</h2>
+     <h2>Worldwide</h2>
+    Huge choirs, rock bands, folk and classical musicians, remote music lessons, in schools and universities,
+    private or public. Even the spoken word — as if you were there in person.
+    <h2>Help?</h2>
     <p>
-      Have a look at the <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">documentation</a>. Especially consider the <a href="wiki/Client-Troubleshooting"
-        target="_blank" rel="noreferrer">troubleshooting section</a>!
+      Check out the <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">documentation</a>. Especially the <a href="wiki/Client-Troubleshooting"
+        target="_blank" rel="noreferrer">troubleshooting</a>!
       You can also ask on the <a href="https://github.com/jamulussoftware/jamulus/discussions" target="_blank" rel="noreferrer">forums</a>.
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Want to get involved?</h2>
+    <h2>Get involved</h2>
     <p>
-    Ideas? Found a bug? Want to contribute some code or help translating into your language? Since Jamulus is <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">free and open source software</a> (FOSS) licensed under the <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html" title="GNU General Public License, version 2" target="_blank" rel="noreferrer">GPL</a>, you can help us!<br/>
-    Take a look at our <a href="wiki/Contribution">contribution guidelines</a> to find out how. Everybody is welcome!
+    Use, see, change, and share — with all. <a href="https://github.com/jamulussoftware">write code</> or <a href="https://hosted.weblate.org/engage/jamulus/">translate it on Weblate</a>.<br/>
+    The <a href="wiki/Contribution">contribution guidelines</a> has more info.
     </p>
     <p>
     <em>
-      For detailed information about how Jamulus hacks the space-time continuum to produce a near-perfect 5th dimension of collaborative sound, see <a
-      href="/PerformingBandRehearsalsontheInternetWithJamulus.pdf">this paper by Volker Fischer (PDF)</a>.
+      Info on the 5th dimension of collaborative sound to be found in <a href="/PerformingBandRehearsalsontheInternetWithJamulus.pdf">this paper by Volker Fischer (PDF)</a>.
```

Comment:
@gilgongo should judge that.

Reply:
It should be evaluated on the merit of having the website scroll for how good of an amusing passage it is.
Having "space-time continuum" there for SEO is misleading.

Reply:
I support the trimming of this far out word jazz.

Reply:
> Having "space-time continuum" there for SEO is misleading.

Agree. But as @gilgongo added it in the first time, I think it's fair to let him judge it.

Reply:
How does "space-time continuum" change SEO patterns?

Reply:
@mcfnerd Having stuff mentioned on the landing page that is not found elsewhere on the site doesn't work well.
This is exemplified in having many entries that all just describe various sub-pages.

Reply:
I've no problem with changing that - it wasn't a particularly considered addition of mine in the first place :-)